### ISSUE TITLE
Does not work

### ISSUE THREAD

**Issue by dumez-k** [oftheheadland/Colorblindly#34]
:)

**Comment by stebogit** [comment]
yeah it seems it doesn't work anymore, I click on the different types and nothing happens on the page, the colors are not affected :(

**Comment by nsjarvis** [comment]
@oftheheadland This formerly awesome plugin needs help, I miss it.  Do you have any idea why it's not working?  I expect it was broken by a chrome update, is it possible to fix it, can we help?

**Comment by nsjarvis** [comment]
Fantastic, thank you!
